packages/rulesets/package.json

```diff
@@ -93,4 +93,4 @@
       "^@stoplight/spectral-ruleset-bundler/(.*)$": "<rootDir>/node_modules/@stoplight/spectral-ruleset-bundler/dist/$1"
     }
   }
-}
```

Comment:
NIT: Restore trailing newline

Reply:
@mikeharder my VS Code appears to remove it on save. Should I reconfigure my VS Code to stop doing this, or is it more about "it is wrong, but let's not make unrelated changes in this PR" ?

Reply:
Our convention is that all files in all repos should end with a trailing newline.  If your VS Code is removing these trailing newlines, then yes it would be best to reconfigure your VS Code.  But I'm a little surprised if VS Code is removing trailing newlines, since ending files with trailing newlines is a pretty widely-used standard.

Reply:
There is nonzero chance I changed the setting myself long time ago, but if I did, I have no recollection of it. I will update my VS Code settings. Thx!